### ISSUE TITLE
Motion to Vacate | Granted | After AddressMotionToVacate, create appeal stream if granted

### ISSUE THREAD

**Issue by leikkisa** [department-of-veterans-affairs/caseflow#13222]
## User or job story
When the Judge addresses a motion to vacate, and it is granted, it should move onto a new appeal stream.

## Acceptance criteria
- [ ] After the AddressMotionToVacateTask, if the judge selects grant, or partial grant, create a new appeal stream
- [ ] Add the original appeal's claimants to the new appeal
- [ ] Add a RootTask
- [ ] Add a JudgeAssignTask, which is a child of RootTask, and assigned to the Judge
- [ ] Clean up code in PostDecisionMotionUpdater that creates other tasks for granted appeals

## Out of scope
- Adding request or decision issues (see #13069)

## Questions
What happens to the additional instructions/context from the AddressMotionToVacateTask?

## Appeal details
Create a copy of the original appeal:
- veteran_file_number: same as original
- legacy_opt_in_approved: same as original
- veteran_is_not_claimant: same as original
- docket_type: same as original
- docket_number: same as original
- type: "Vacate"

## Release notes
N/A

**Comment by leikkisa** [comment]
Blocking currently until we learn whether denials/dismissals should get their own appeal stream, since in the current flow they do not get a decision document.

If they do _not_ get a new appeal stream, the contents of this ticket will be replaced by starting the new appeal stream only after AddressMotionToVacate.

**Comment by leikkisa** [comment]
Moved to current sprint since due to priority, but we marked it as not originally being pulled in.
